Jenkinsfile

```diff
@@ -39,9 +39,21 @@ onlyOnMaster {
     }
 
     milestone()
-    stage("prod deploy") {
-        input(message: "Deploy to prod?")
-        milestone()
-        deployApp(image: img, app: "h-periodic", env: "prod", region: "us-west-1")
+    stage("approval") {
+        input(message: "Proceed to production deploy?")
+    }
+
+    milestone()
+    stage("prod Deploy") {
+        parallel(
+            us: {
+                deployApp(image: img, app: "h-periodic", env: "prod", region: "us-west-1")
+            },
+            ca: {
+		// Workaround to ensure all parallel builds happen. See https://hypothes-is.slack.com/archives/CR3E3S7K8/p1625041642057400
+                sleep 2
+                deployApp(image: img, app: "h-periodic-ca", env: "prod", region: "ca-central-1")
+            }
+        )
```

Comment:
Parallel block introduced to enable multi-region deploys for production environments. Here you will see steps for: `us` and `ca`.